### DISCUSSION TITLE
Offtopic: What terminal color scheme are you using?

### DISCUSSION THREAD

**Discussion by adrian5** [Siilikuin/minimum-viable-hugo#6] [General]
Hello Andrew, sorry to bother you here but I was wondering what color scheme you're using in your screenshots:

![](https://user-images.githubusercontent.com/53230903/221623209-8a557a70-b81a-4232-a067-b3b0f1f1c577.png)

I read your article on fzf which had several code gifs that I think showed the same theme, but the website is down atm.

Thanks!

**Comment by adrian5** [comment]
Now that the site is back up, this theme I mean:

![](https://andrew-quinn.me/img/fzf-ctrl-r.gif)

**Comment by hiAndrewQuinn** [comment]
Oh Jesus, I don't have a clue to be honest... What I _can_ tell you is that

- The OS is standard Ubuntu,
- The window manager (for those thin blue borders) is [default i3wm](https://i3wm.org/),
- The terminal emulator is [kitty](https://sw.kovidgoyal.net/kitty/conf/), which I chose because it was the only terminal emulator I could find that supported font ligatures natively (image below),
- The font is Caskaydia Code, which is itself a clone of the Cascadia Code font in Windows Terminal (because - gasp! - I use Windows at work!),
-  [tmux](https://github.com/tmux/tmux) is providing the green line at the bottom and around the edges, and
- The shell is `zsh` with whatever the standard `oh-my-zsh` is.

I think all of the defaults of those blended together to create whatever delighted your eyes there. Oh, and the thin green border and green bottom line come from standard. I actually only rarely customize the colors themselves

![image](https://user-images.githubusercontent.com/53230903/227500303-55750374-8c53-4693-8aa3-3de6abfb0d7c.png)

**Reply by hiAndrewQuinn** [reply]
^Those are just examples of ligatures, Caskaydia looks a little meatier imo.

**Comment by adrian5** [comment]
Thank you for the detailed response Andrew!

Using that info I did some digging and it appears the colors are from Kitty's default theme ([man page](https://man.archlinux.org/man/community/kitty/kitty.conf.5.en#THE_COLOR_TABLE)):

****Regular****

black:   `#000000`
red:     `#cc0403`
green:   `#19cb00`
yellow:  `#cecb00`
blue:    `#0d73cc`
magenta: `#cb1ed1`
cyan:    `#0dcdcd`
white:   `#dddddd`

****Bright****

black:   `#767676`
red:     `#f2201f`
green:   `#23fd00`
yellow:  `#fffd00`
blue:    `#1a8fff`
magenta: `#fd28ff`
cyan:    `#14ffff`
white:   `#ffffff`
